### PR TITLE
Contract interaction docs

### PR DESCRIPTION
**Pull request by arthcp**
- Move docs to interact with pos contracts using Matic sdk in a separate folder.
- Add docs to interact by directly calling pos contracts.
- Update deployment details.
- Remove some unused files.

### CODE REVIEW

File: docs/develop/ethereum-matic/pos/using-sdk/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```js
+const MaticPOSClient = require('@maticnetwork/maticjs').MaticPOSClient
+
+// or
+
+import { MaticPOSClient } from '@maticnetwork/maticjs'
+```
+
+Instantiate the client by passing network details and providers based on the network you are integrating. The providers can be RPC urls, Metamask provider, HDWalletProvider etc. based on requirement.
+```jsx
```
Comment (itzmeanjan) [inline docs/develop/ethereum-matic/pos/using-sdk/getting-started.md:29]: We can put them under seperate tabs ( using `<TabItem>` ) . I think that looks better.

**Review by itzmeanjan — COMMENTED** [review]
Let me know your thoughts ;)

File: docs/develop/ethereum-matic/pos/calling-contracts/erc1155.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+1. **_Burn_** tokens on matic chain.
+2. Call **_exit_** function on **_RootChainManager_** to submit proof of burn transaction. This call can be made **_after checkpoint_** is submitted for the block containing burn transaction.
+
+## Step Details
+---
+
+### Instantiate the contracts
+```js
+const mainWeb3 = new Web3(mainProvider)
```
[the remark this answers is not in the document]
Comment (arthcp) [reply docs/develop/ethereum-matic/pos/calling-contracts/erc1155.md:29]: I think this is self explanatory. And we expect only advanced users to call contracts directly so it should be enough.

**Comment by arthcp** [comment]
Addresses the comments. Sorry for late response. :)

File: docs/develop/ethereum-matic/pos/using-sdk/getting-started.md (hunk 1 above)
Comment (arthcp) [reply docs/develop/ethereum-matic/pos/using-sdk/getting-started.md]: Did the changes.

File: docs/develop/ethereum-matic/pos/calling-contracts/erc1155.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (itzmeanjan) [reply docs/develop/ethereum-matic/pos/calling-contracts/erc1155.md:29]: Hm, you're correct :+1:

**Review by itzmeanjan — COMMENTED** [review]
Looks fine, we can go for merging :+1:

**Pull request merged**